### ISSUE TITLE
Store picker: tapping Dismiss still switches to another store

### ISSUE THREAD

**Issue by jaclync** [woocommerce/woocommerce-ios#5246]
**Describe the bug**

After selecting a different store in the store picker, tapping Dismiss still switches to another store.

**To Reproduce**

Prerequisite: the account is connected to more than one store

- Go to settings > Switch Store
- Select a different store
- Tap Dismiss in the navigation bar --> the app switches to the selected store in the previous step

**Expected behavior**
The app should not switch stores because the user intends to dismiss the store picker.

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: any
- iOS version: 
- WooCommerce iOS version: 7.7 (but I think this is a pre-existing issue)

**Comment by jaclync** [comment]
Duplicate of https://github.com/woocommerce/woocommerce-ios/issues/3834, closing this!
